microsite/data/plugins/codescene.yaml

```diff
@@ -0,0 +1,9 @@
+---
+title: CodeScene
+author: Empear
```

Comment:
We recently changed company name to "CodeScene". Can you please replace "Empear" with "CodeScene"?

Reply:
Yup, done